rediscluster/pipeline.py

```diff
@@ -142,144 +146,70 @@ def send_cluster_commands(self, stack, raise_on_error=True, allow_redirections=T
         `allow_redirections` If the pipeline should follow `ASK` & `MOVED` responses
         automatically. If set to false it will raise RedisClusterException.
         """
-        if self.refresh_table_asap:
```

Comment:
After removing this part the variable `self.refersh_table_asap` is not in use in the class so the variable is not needed in the `__init__` method. The refresh table logic is still in use in the regular client class.